diagnostic.rb

```diff
@@ -2,10 +2,23 @@
 # Diagnostic answers to:
 
 # question 2:
-
+Licensee.create(surname:"Bishop", given_name:"Caren", gender:"f", height:63, weight:132, born_on:"1943-09-26", eye_color:"Brown", hair_color:"Black")
 # question 3:
-Licensee.transaction do
-  CSV.foreach 'data/licensees.csv', headers: true do |row|
+namespace :db do
+  namespace :populate do
+    desc 'Populate database with example data'
+    task all: [:licensees]
+
+    desc 'Populate database with licensees'
+    task licensees: :environment do
+      Licensee.transaction do
+        CSV.foreach(Rails.root + 'data/licensees.csv',
+                    headers: true) do |row|
+                      licensee = row.to_hash
+                      Licensee.create! licensee unless Licensee.exists? licensee
+                    end
+      end
+    end
```

Comment:
It wasn't strictly necessary to write this as a `rake` task, btw - you could have just written those two lines inside the `do` block.